### PR TITLE
fix cookie path in cookieStorage

### PR DESCRIPTION
**Pull request by nanxiaobei**
If the path of cookie is not set, there will be multiple `wagmi.store` in the cookie (under different paths), and the values are inconsistent. At this point, obtaining the value of the cookie will be uncontrollable.

### CODE REVIEW

**Comment by nanxiaobei** [comment]
vSnbR5d

File: packages/core/src/utils/cookie.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ export const cookieStorage = {
   },
   setItem(key, value) {
     if (typeof window === 'undefined') return
-    document.cookie = `${key}=${value}`
+    document.cookie = `${key}=${value};path=`
```
Comment (tmm) [inline packages/core/src/utils/cookie.ts:13]: what's the advantage of `path=` over `path=/`?

**Review by tmm — CHANGES_REQUESTED** [review]
Please add a patch changeset with `pnpm changeset`.

File: packages/core/src/utils/cookie.ts (hunk 1 above)
Comment (nanxiaobei) [reply packages/core/src/utils/cookie.ts]: It's a mistake... fixed.

**Pull request merged**